textrepo-app/src/main/java/nl/knaw/huc/resources/rest/ContentsResource.java

```diff
@@ -42,11 +43,16 @@ public ContentsResource(ContentsService contentsService, ContentsHelper contents
   @Path("/{sha}")
   @Timed
   @Produces({APPLICATION_OCTET_STREAM, APPLICATION_JSON})
-  @ApiOperation(value = "Retrieve contents")
-  @ApiResponses(value = {@ApiResponse(code = 200, response = byte[].class, message = "OK")})
+  @ApiOperation(value = "Retrieve contents as file")
+  @ApiResponses(value = {@ApiResponse(code = 200, response = String.class, message = "OK")})
   public Response getContents(
-      @HeaderParam(ACCEPT_ENCODING) String acceptEncoding,
-      @PathParam("sha") @NotBlank String sha
+      @ApiParam(allowableValues = "gzip")
```

Comment:
Actually .... {"gzip", "x-gzip", "deflate, gzip", "x-gzip, compress"} are all allowed here (according to some spec.)

as long as it *contains* "gzip" it will work for us.

Same further down.

Reply:
`allowableValues` must be a string, which can be a `comma-separated list`, but our possible gzip-values already contain comma's, and escaping the comma's, or marking the whole string with quotes doesn't work.
For now we'll keep it at `"gzip"`